stock-plugins/aiverify.stock.accumulated-local-effect/algorithms/tests/e2e/__main__.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+from test_engine_core.plugins.enums.model_type import ModelType
+from accumulated_local_effect.algo_init import AlgoInit
+
+def run():
+    test_accumulated_local_effect_plugin()
+
+def test_accumulated_local_effect_plugin():
+    # # Binary Classification Non-Pipeline
+
+    # data_path = str( Path().absolute() / "../../test-assets/data/sample_bc_credit_data.sav")
+    # model_path = str(Path().absolute() / "../../test-assets/model/sample_bc_credit_sklearn_linear.LogisticRegression.sav")
+    # ground_truth_path = str( Path().absolute() / "../../test-assets/data/sample_bc_credit_data.sav")
+    # ground_truth = "default"
+    # run_pipeline = False
+    # model_type = ModelType.CLASSIFICATION
+
+    # # Multiclass Classification Non-Pipeline
```

Comment:
If all the tests work and are expected to work, let's run this as 4 different tests instead. Take a look at `pytest.mark.parametrize`